tests/test_rsa

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
What is the value of changing key file name?

Reply:
1. I find it a good rule to name any publicized ssh key pairs as 'test' or 'example' or 'sample' to make sure it's clear this isn't meant to be used for stuff.
2. I'm no longer creating a 'kubevirt' user, so don't want the key named 'kubevirt'.

Reply:
and what is the reason for not creating the user.